src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final double NUMBER_TEN = 10.0;
```

Comment:
the name looks strange) let's rename it
same for other figure with the same constant